modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -916,16 +912,16 @@ open class PlaybackManager @Inject constructor(
     suspend fun onBufferingStateChanged() {
         pocketCastsPlayer?.let {
             val isBuffering = it.isBuffering()
-            val bufferedMs = it.bufferedUpToMs()
             withContext(Dispatchers.Main) {
+                val bufferedMs = it.bufferedPosition
```

Comment:
When the `PlaybackManager` gets the buffered position from the player, we now have to make sure we're on the main thread.